resources/views/profile/delete-user-form.blade.php

```diff
@@ -21,9 +21,9 @@
             </x-slot>
 
             <x-slot name="description">
-                <div class="flex flex-col mt-4">
+                <div class="flex flex-col">
                     <div class="flex justify-center w-full">
-                        <x-ark-icon name="fortify-modal.delete-account" class="w-2/3 h-auto text-theme-primary-600"/>
+                        <x-ark-icon name="fortify-modal.delete-account" class="my-8 w-60 h-auto text-theme-primary-600"/>
```

Comment:
🤔 the space between the image and the following component should be 32px, but since the alert component have 16px margin-top (mt-4), the space results 48px. The same for the description text.
```suggestion
                        <x-ark-icon name="fortify-modal.delete-account" class="mt-8 mb-4 w-60 h-auto text-theme-primary-600"/>
```